source/components-and-apis.rst

```diff
@@ -168,43 +171,73 @@ which component endpoint.
+.. _readthedocs - auth: http://dojotdocs.readthedocs.io/projects/auth/en/latest/
 
-.. _GitHub - iotagent-json: https://github.com/dojot/iotagent-json
-.. _readthedocs - iotagent-json: http://dojotdocs.readthedocs.io/projects/iotagent-json/en/latest/
+.. _GitHub - history-ws: https://github.com/dojot/history-ws
+.. _API - history-ws: https://dojot.github.io/history-ws/apiary_latest.html
 
-.. _GitHub - Orion: https://github.com/dojot/fiware-orion
-.. _readthedocs - Orion: http://fiware-orion.readthedocs.io/en/latest/
-.. _Orion v1: http://telefonicaid.github.io/fiware-orion/api/v1/
-.. _Orion v2: http://telefonicaid.github.io/fiware-orion/api/v2/stable/
 
 .. _GitHub - DeviceManager: https://github.com/dojot/device-manager
-.. _API - DeviceManager: http://dojotdocs.readthedocs.io/projects/DeviceManager/en/latest/api.html
+.. _API - DeviceManager: https://dojot.github.io/device-manager/apiary_latest.html
 .. _readthedocs - DeviceManager: http://dojotdocs.readthedocs.io/projects/DeviceManager/en/latest/
+.. _Messages - DeviceManager: http://dojotdocs.readthedocs.io/projects/DeviceManager/en/latest/kafka-messages.html
 
-.. _GitHub - auth: https://github.com/dojot/auth
-.. _API - auth: https://dojot.github.io/auth/apis.html
+.. _GitHub - image-manager: <https://github.com/dojot/image-manager>
+.. _API - image-manager: https://dojot.github.io/image-manager/apiary_latest.html
+
+
+.. _GitHub - GUI: https://github.com/dojot/gui
+
+
+.. _GitHub - flowbroker: https://github.com/dojot/flowbroker
+.. _API - flowbroker: https://dojot.github.io/flowbroker/apiary_latest.html
+
+.. _GitHub - data-broker: https://github.com/dojot/data-broker
+.. _API - data-broker: https://dojot.github.io/data-broker/apiary_latest.html
+
+
+.. _GitHub - iotagent-json: https://github.com/dojot/iotagent-json
+.. _readthedocs - iotagent-json: http://dojotdocs.readthedocs.io/projects/iotagent-json/en/latest/
+.. _Messages - iotagent-json: http://dojotdocs.readthedocs.io/projects/iotagent-json/en/latest/operation.html#sending-messages-to-other-components-via-kafka
 
-.. _GitHub - perseo-core: https://github.com/telefonicaid/perseo-core
-.. _Docs - perseo-core: https://github.com/telefonicaid/perseo-core/tree/master/documentation
+.. _GitHub - iotagent-mosca: https://github.com/dojot/iotagent-mosca
 
-.. _GitHub - perseo-fe: https://github.com/dojot/perseo-fe
-.. _Docs - perseo-fe: https://github.com/dojot/perseo-fe/tree/cpqd_master/documentation
+.. _GitHub - data-broker: https://github.com/dojot/data-broker
 
 .. _GitHub - mashup: https://github.com/dojot/mashup
 
-.. _GitHub - EJBCA-REST: http://dojotdocs.readthedocs.io/projects/EJBCA-REST/en/latest/
-.. _API - EJBCA-REST: http://dojotdocs.readthedocs.io/projects/EJBCA-REST/en/latest/api.html
+.. _GitHub - EJBCA-REST: https://github.com/dojot/ejbca-rest
+.. _API - EJBCA-REST: https://dojot.github.io/ejbca-rest/apiary_latest.html
 
+.. _GitHub - alarm-manager: https://github.com/dojot/alarm-manager
```

Comment:
Kerberos/ MA are missing

Reply:
They are not currently included in docker-compose and they are not yet thaaaaat stable.